improver_tests/blending/blend_across_adjacent_points/test_TriangularWeightedBlendAcrossAdjacentPoints.py

```diff
@@ -53,11 +53,13 @@ def set_up_cubes_for_process_tests():
     central_cube = set_up_variable_cube(
         np.array([[1., 1.], [1., 1.]], dtype=np.float32),
         name='lwe_thickness_of_precipitation_amount', units='m',
-        time=dt(2017, 1, 10, 3, 0), frt=dt(2017, 1, 10, 3, 0))
+        time=dt(2017, 1, 10, 3, 0), frt=dt(2017, 1, 10, 3, 0),
+        time_bounds=(dt(2017, 1, 10, 1, 0), dt(2017, 1, 10, 4, 0)))
```

Comment:
OK this is picky but in all cases the IMPROVER convention is that the time point should be equal to the upper bound.  So could you make these match please (for both cubes).